cmd/plume/prune.go

```diff
@@ -56,6 +67,18 @@ func runPrune(cmd *cobra.Command, args []string) {
 	if len(args) > 0 {
 		plog.Fatal("No args accepted")
 	}
+	if daysLastLaunched < 0 {
```

Comment:
I think I would rather keep the if-blocks separate so that we can report a clear error messages to users. But you made me realize I need at least one more check: `if !checkLastLaunched && daysLastLaunched > 0` so that we tell users that we report that the option will be ignored.